tests/test_minimal/test_tools/test_expand_paths.py

```diff
@@ -380,3 +382,96 @@ def test_expand_paths_ibl(tmpdir):
 
     tc = unittest.TestCase()
     tc.assertCountEqual(path_expansion_results, expected)
+
+
+# contruct_path_template tests
+
+
+test_cases = [
+    dict(
+        test_input=dict(
+            path="/data/subject456/session123/file.txt", metadata=dict(subject_id="subject456", session_id="session123")
+        ),
+        expected="/data/{subject_id}/{session_id}/file.txt",
+    ),
+    dict(
+        test_input=dict(
+            path="/data/subject789/session456/image.txt",
+            metadata=dict(subject_id="subject789", session_id="session456", file_type="image"),
+        ),
+        expected="/data/{subject_id}/{session_id}/{file_type}.txt",
+    ),
+    dict(
+        test_input=dict(
+            path="/SenzaiY/YMV01/YMV01_170818/YMV01_170818.eeg", metadata=dict(session_id="170818", subject_id="YMV01")
+        ),
+        expected="/SenzaiY/{subject_id}/{subject_id}_{session_id}/{subject_id}_{session_id}.eeg",
+    ),
+    dict(
+        test_input=dict(path="/SenzaiY/YMV01/YMV01_170818/", metadata=dict(session_id="170818", subject_id="YMV01")),
+        expected="/SenzaiY/{subject_id}/{subject_id}_{session_id}/",
+    ),
+    dict(
+        test_input=dict(
+            path="/steinmetzlab/Subjects/NR_0017/2022-03-22/001/raw_video_data/_iblrig_leftCamera.raw.6252a2f0-c10f-4e49-b085-75749ba29c35.mp4",
```

Comment:
@CodyCBakerPhD this is a tricky one. What would you consider the session_id? "2022-03-22/001"?

Reply:
It could be `2022-03-22`, or `2022-03-22/001`, or the UUID before the MP4 file `6252a2f0-c10f-4e49-b085-75749ba29c35`